src/annotator/util/test/listener-collection-test.js

```diff
@@ -0,0 +1,30 @@
+import { ListenerCollection } from '../../util/listener-collection';
+
+describe('ListenerCollection', () => {
+  let listeners;
+
+  beforeEach(() => {
+    listeners = new ListenerCollection();
+  });
+
+  afterEach(() => {
+    listeners.removeAll();
+  });
+
+  it('registers and triggers event listener', () => {
+    const listener = sinon.stub();
+    listeners.add(window, 'resize', listener);
+
+    window.dispatchEvent(new Event('resize'));
+    assert.calledOnce(listener);
+  });
+
+  it('unregister event listeners', () => {
+    const listener = sinon.stub();
+    listeners.add(window, 'resize', listener);
+    listeners.removeAll();
```

Comment:
I think it would be good to add a couple of listeners in this test, to verify that _all_ of them are removed.